app/models/comment.rb

```diff
@@ -1,2 +1,5 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :text, presence: true, length: { minimum: 5, maximum: 1000 }
```

Comment:
- [ ]  It should declare that this comment belongs to an Author, **hint (belong_to => author and class_name to the user);**
- [ ] It should update_comments_counter method increment the number of comments for the given post.
The code will increment the counter for all posts that belong to the author with a username of "Joe".
The code will increment the counter for all posts that belong to the author with a username of "Joe" and also increment it by one.**hint (post.increment => comments_counter)**